test/ret_test.exs

```diff
@@ -334,6 +334,21 @@ defmodule RetTest do
       assert admin_account.account_id === updated_scene.account_id
       assert scene.scene_id === updated_scene_listing.scene_id
     end
+
+    test "deletes owned files" do
+      {:ok, admin_account: admin_account} = create_admin_account("admin")
+      target_account = create_account("target")
+
+      owned_files = 1..3 |> Enum.map(fn _ -> generate_temp_owned_file(target_account) end)
```

Comment:
It might be nice to extract a private function:
```elixir
@spec generate_owned_files(Account.t()) :: Stream.t()
defp generate_owned_files(%Account{} = account),
  do: Stream.repeatedly(fn -> generate_temp_owned_file(target_account) end)
```

You could use it here like so:
```elixir
owned_files =
  target_account
  |> generate_owned_files()
  |> Enum.take(3)
```

and down in `create_scene/2`, `create_asset/1`, and `create_avatar/1`.

Reply:
Yup, covered in #626